arch/risc-v/src/c906/c906_timerisr.c

```diff
@@ -117,15 +64,7 @@ static int c906_timerisr(int irq, void *context, void *arg)
 
 void up_timer_initialize(void)
 {
-  /* Attach timer interrupt handler */
-
-  irq_attach(RISCV_IRQ_MTIMER, c906_timerisr, NULL);
-
-  /* Reload CLINT mtimecmp */
-
-  c906_reload_mtimecmp();
-
-  /* And enable the timer interrupt */
-
-  up_enable_irq(RISCV_IRQ_MTIMER);
+  up_alarm_set_lowerhalf(riscv_mtimer_initialize(
```

Comment:
can we use temp variable here, just for easy reading?

Reply:
OK